test.sh

```diff
@@ -19,6 +16,6 @@ else
 fi
 
 docker run -d --name cypress-test cypress-visual-regression sleep infinity
-docker exec cypress-test bash -c './node_modules/.bin/cypress run --env type=base'
-docker exec cypress-test bash -c './node_modules/.bin/cypress run --env type=actual'
```

Comment:
Note here: I wanted also to put all plugin env variables under one object, so that you can configure it as:
{
  env: {
    visualRegression: {
      ... all config here ...
    }
}

but that screws up this kinds of commands, which people might rely on

Reply:
tbh I would go for both and have a big release right away, but we could set old properties as deprecated and support them till next big release, what do you think? More work, but more safe and encourages big changes right away

Reply:
Yea at least mark it as deprecated but not remove it because right now we  are using the env type 🥵 in our projects.